script.js

```diff
@@ -0,0 +1,12 @@
+const humburger = document.getElementById('hamburger');
+const navLinks = document.getElementById('navLinks');
+humburger.addEventListener('click', () => {
+  navLinks.classList.toggle('show');
+});
+navLinks.addEventListener('click', (event) => {
+  if (navLinks.classList.contains('show')) {
+    if (event.target.tagName === 'A' || event.target.tagName === 'I') {
+      navLinks.classList.remove('show');
+    }
+  }
```

Comment:
- [ Required ] I think it would be better if you made it possible for a user to scroll to the section they click from the menu. An example of this would be if I click on `Contact` link, the contact section should be displayed and if I click on the `About` the about section should be displayed. Currently, when I click on a link nothing happens, kindly rectify this.